### DISCUSSION TITLE
Design Improvement Suggestions - Quick & Dirty Fixes

### DISCUSSION THREAD

**Discussion by trashpandaqc** [datafruits/datafruits#1349] [Ideas]
Some ideas that don't depend on a total overhaul, add more?

Menu bar reoganization:
  -  start with "radio" section expanded, whether the homepage is chat or stays as is
  -  move chat into radio section
  -  make an expandable "more" section and move About, support, dj on datafruits, fruit standz, fruitopiyeah, and shrimpo in there
  -  remove the 3D button for now 😢
  -  put a "?" icon on the right with...

Help feature
- site map / tutorial / tool tips / a screenshot of the page with some arrows and explanations scribbed on?

Make chat the homepage and or splitscreen with the current one

**Comment by scoggs** [comment]
Paul Manton: instead of designing some new help / tooltip feature on top of what we have maybe we should consider renaming sections and re-arranging the menu -- it could be a lot less work with a similar outcome

**Comment by dudethatbe** [comment]
As far as renaming menu labels go, Caius had a good suggestion in an earlier thread https://github.com/datafruits/datafruits/discussions/1342#discussion-6765172

> 
    Chat
    Radio
        Timetable
        Shows
        Archives
        DJ on Datafruits
    Community
        Fruits
        SHRIMPO
        GitHub/Hack
        da fruit standz (Forum)
        Fruitopiyeah (Wiki)
    Stuff
        Label
        Shop
    About
    Support

I'd worry that putting Chat in Radio would make it even harder for new users to find so maybe we really should just make that the homepage. But I really like the idea of having the Radio menu already expanded @trashpandaqc 

More saliently, having a help feature is what inspired this thread in the first place. The suggestion from @scoggs is a valid one; There'd be less to explain if the labels were easier to understand. We have added aria labels to (mostly) everything, but that doesn't give enough space to accurately describe what a funny-looking button with a dancing fruit does. 

I think having a static image toggle on/off is fine if the site is full-screened on desktop, but then you have to think about mobile, and how the site automatically readjusts to a browser window thats smaller than fullscreen. Maybe we can have a tool-tip system that looks like the modal dialogs? Maybe the proposed **?** button for the nav bar would change the cursor and enable the "help mode" or whatever 😇

As for the easier requests like removing the 3D button, an issue can be created and it should be relatively easy to get that merged. Thanks for the ideas so far everyone!
